src/base/bittorrent/session.cpp

```diff
@@ -1710,7 +1720,12 @@ void Session::handleTorrentFinished(TorrentHandle *const torrent)
 void Session::handleTorrentResumeDataReady(TorrentHandle *const torrent, const libtorrent::entry &data)
 {
     --m_numResumeData;
-    writeResumeDataFile(torrent, data);
+
+    QByteArray out;
+    libt::bencode(std::back_inserter(out), data);
```

Comment:
Please tell me what to write here (not to put here my bad English).
>Especially the 2nd one makes sense. 

The first is no less important. When you perform file operations computing resources are idle, so it makes sense to do something parallel to this.